packages/@webex/plugin-meetings/src/member/util.ts

```diff
@@ -347,6 +353,22 @@ MemberUtil.extractId = (participant: any) => {
   return null;
 };
 
+/**
+ * extracts the media status from nested participant object
+ * @param {Object} participant the locus participant
+ * @returns {Object}
+ */
+MemberUtil.extractMediaStatus = (participant: any): IMediaStatus => {
```

Comment:
Not sure if we are following this in js-sdk but when the code is in ts, it's better to avoid using any as type. See if it's possible to specify appropriate type here for participant

Reply:
participant object comes as a response of locus update event and not declared in any file. It will be little tricky to get all the types of properties contained in participant object. Also, this looks more of a refactor thing which is out of scope of this ticket.